cmd/lotus-storage-miner/main.go

```diff
@@ -32,6 +32,7 @@ func main() {
 		initCmd,
 		runCmd,
 		stopCmd,
+		configCmd,
```

Comment:
We'll likely rename this to `config default` or something like that, when we'll be refactoring the config system

Fine for now though

Reply:
I was taking inspiration from `telegraf` and `influx` which have this same config pattern